internal/ingress/controller/nginx.go

```diff
@@ -343,17 +343,10 @@ func (n *NGINXController) Start() {
 			// issues because of this behavior.
 			// To avoid this issue we restart nginx in case of errors.
```

Comment:
@aledbf do I understand correctly, now if master process dies, and controller detects it, controller will no longer try to restart master process but it will exit as well?

If so, these comments need to be updated.

Reply:
@sslavic done https://github.com/kubernetes/ingress-nginx/pull/5760